test/common/components/SampleFeature.unit.spec.jsx

```diff
@@ -7,21 +7,17 @@ import SampleFeature from '../../../src/js/common/components/SampleFeature.jsx';
 describe('<SampleFeature>', () => {
   it('should be empty when not enabled', () => {
     const tree = SkinDeep.shallowRender(
-      <div>
-        <SampleFeature/>
-      </div>
+      <SampleFeature/>
```

Comment:
This is perhaps a result of their introducing the [dive](https://www.npmjs.com/package/skin-deep#going-deeper) command?